unittests/tools/test_bundler_audit_parser.py

```diff
@@ -56,3 +56,11 @@ def test_get_findings_version9(self):
                 self.assertEqual("CVE-2018-3760", finding.unsaved_vulnerability_ids[0])
                 self.assertEqual("sprockets", finding.component_name)
                 self.assertEqual("2.2.3", finding.component_version)
+            with self.subTest(i=3):
+                finding = findings[3]
+                self.assertEqual("Gem nokogiri: Improper Handling of Unexpected Data Type in Nokogiri [GHSA-xc9x-jj77-9p9j]", finding.title)
+                self.assertEqual("Medium", finding.severity)
+                self.assertEqual(1, len(finding.unsaved_vulnerability_ids))
+                self.assertEqual("GHSA-xc9x-jj77-9p9j", finding.unsaved_vulnerability_ids[0])
+                self.assertEqual("nokogiri", finding.component_name)
+                self.assertEqual("1.15.2", finding.component_version)
```

Comment:
Could you also please add then another unittest for "finding.cve"?

Reply:
https://github.com/DefectDojo/django-DefectDojo/pull/9649/commits/626defd50904b720429a8dd37f0b7891242ee014